tests/testthat/test-derive_vars_dt.R

```diff
@@ -292,6 +292,71 @@ test_that("compute_dtf Test 13: compute DTF", {
   )
 })
 
+# restrict_imputed_dtc_dt ----
+test_that("", {
```

Comment:
Test Description please - unless there is reason??

Reply:
No this was a complete oversight, good catch